packages/server/src/asyncQuestion/asyncQuestionVotes.entity.ts

```diff
@@ -0,0 +1,36 @@
+import { Exclude } from 'class-transformer';
+import { UserModel } from '../profile/user.entity';
+import {
+  BaseEntity,
+  Column,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { AsyncQuestionModel } from './asyncQuestion.entity';
+
+@Entity('async_question_votes_model')
+export class AsyncQuestionVotesModel extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @ManyToOne((type) => UserModel)
+  @JoinColumn({ name: 'userId' })
+  user: UserModel;
+
+  @Column({ nullable: false })
+  @Exclude()
+  userId: number;
+
+  @ManyToOne((type) => AsyncQuestionModel)
+  @JoinColumn({ name: 'questionId' })
+  question: AsyncQuestionModel;
+
+  @Column({ nullable: false })
+  @Exclude()
+  questionId: number;
+
+  @Column({ nullable: false })
+  vote: number;
```

Comment:
Maybe enum would be more explanatory?


```
export enum VoteType {
  UPVOTE = 'upvote',
  DOWNVOTE = 'downvote',
}
```

Reply:
This column represents the sum of all the votes on a question from a user. It needs to be a number. This number is incremented/decremented whenever a user votes on a question.